test/docker-compose-with-error.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
 
+# You can pass TARGETS to this script to specify
+# which services to run, instead of all of them, which is the default behavior.
+TARGETS="${TARGETS}"
```

Comment:
`TARGETS="${TARGETS}"` seems redundant, I'd think we can remove but keep the comment. Your call tho.